recipes/app2.rb

```diff
@@ -30,20 +30,29 @@
 
 #### Apps ####
 
-# this app's service depends on the logs/ directory being present inside
-# ~formsender-staging/
-osl_app 'formsender-staging-gunicorn' do
-  description 'formsender staging app'
-  user 'formsender-staging'
-  start_cmd '/home/formsender-staging/venv/bin/gunicorn -b 0.0.0.0:8086 '\
-    '-D --pid /home/formsender-staging/tmp/pids/gunicorn.pid '\
-    '--access-logfile /home/formsender-staging/logs/access.log '\
-    '--error-logfile /home/formsender-staging/logs/error.log '\
-    '--log-level debug '\
-    'formsender.wsgi:application'
-  environment 'PATH=/home/formsender-staging/venv/bin'
-  working_directory '/home/formsender-staging/formsender'
-  pid_file '/home/formsender-staging/tmp/pids/gunicorn.pid'
+directory '/formsender'
+
+git '/var/lib/formsender/' do
+  repository 'https://github.com/osuosl/formsender.git'
+  revision 'antoniagaete/RT_api'
+  action :sync
+end
```

Comment:
I believe `:sync` is the default action, so please remove. Also remove the trailing slash.
```suggestion
git '/var/lib/formsender' do
  repository 'https://github.com/osuosl/formsender.git'
  revision 'antoniagaete/RT_api'
end
```